_posts/2023-08-15-OLS-joining-forces-with-communities-to-broaden-participation-in-open-science.md

```diff
@@ -0,0 +1,97 @@
+
+**Authors:** Laurah Nyasita Ondari, Laura Acion, Nicolás Palopoli, Malvika Sharan, Andela Van de Valt, Lena Karvovskaya, Laura Ascenzi, Esther Plomp, Mike Landi Kofia and Paz Bernaldo.
+
+[OLS](https://openlifesci.org/) is joining forces with the [Bioinformatics Hub of Kenya initiative (BHKi)](https://bhki.org/), the South African Center for Digital Language Resources [(SADiLaR)](https://sadilar.org/index.php/en/) via the [ESCALATOR programme](https://escalator.sadilar.org/), [VU Amsterdam](https://vu.nl/en) and [MetaDocencia](https://www.metadocencia.org/), to bring together the newest [Open Seeds cohort](https://openlifesci.org/posts/2023/04/18/Introducing-the-new-OLS/), OLS-8. Our goal is to connect community members from our respective regions in the international open science network, as well as strengthen their role as ambassadors for open science through local activities. 
+
+Leads and members from the partnering organisations and communities have worked with OLS since the very beginning. As OLS fellows, they have taken different roles in Open Seeds as well as provided the OLS team with several opportunities to learn from their work. 
+
+**Who we are:**
+
+- Mike Landi Kofia and Laurah Nyasita (representing BHKi), and Anelda Van der Walt (representing ESCALATOR by SADILAR) are piloting OLS-Africa chapter under which they will upskill undergraduate, postgraduate, early- and mid-career researchers in open science as well as create local opportunities in Kenya and South Africa.
+
+- Lena Karvovskaya is leading an Open Science Champions programme in VU Amsterdam for researchers and support staff, successful candidates for which will undergo the Open Seeds programme and engage through open communication efforts locally in Amsterdam. 
+
+- Laura Acion and Nicolás Palopoli, co-directors of MetaDocencia, are engaging their team to learn the operational aspects of Open Seeds to plan and deliver cohort-based training for NASA-TOPS, as well as share their learnings with the broader community. 
+
+Open Seeds Programme Coordinator, Paz Bernaldo, the OLS Directors and team members will contribute to ensure that OLS-8 is delivered as planned. Likewise, OLS's existing collaborations, with TU Delft, [The Turing Way](https://the-turing-way.netlify.app/index.html), [ELIXIR](https://elixir-europe.org/) and [EOSC-Life](https://www.eosc-life.eu/), have been instrumental and will continue to be so in OLS-8. As an example, the [Faculty of Applied Sciences at TU Delft](https://www.tudelft.nl/en/faculty-of-applied-sciences/about-faculty), thanks to its Data Steward, Esther Plomp, has collaborated with Open Seeds since OLS-4 (2021), ensuring that participating PhD candidates [are eligible for 5 Discipline-related credits for their Doctoral Education Programme](https://estherplomp.github.io/TNW-OS-support/posts/open-science-OLS/).
+
+Below we share the goals and visions behind these new partnerships and explain what they mean for the OLS-8 cohort.
+## OLS-Africa
```

Comment:
```suggestion

## OLS-Africa

```

Reply:
oh Yo had the idea of using Africa-OLS and MD-OLS instead of putting us first hehe